server/routes/mentions.js

```diff
@@ -56,4 +56,19 @@ router.get("/toggle", async (req, res) => {
   res.json({ message: "Endpoint for mention toggles" });
 });
 
+router.get("/:idAndPlatform", async (req, res) => {
```

Comment:
Hmm consider modifying the resource endpoint just a tiiiny bit; maybe make it `/by-id-platform/:idAndPlatform`. I think I have to also change the names of some of my endpoints.

The reason for doing so is because right now you have that resource requiring req.params at the root of `/api/mentions/`: suppose you have multiple endpoints that check for a param in `req.params` (maybe one is something called like `/api/mentions/:justById`). 

Correct me if I'm wrong, but I believe the way express parses endpoints is that it doesn't actually know the name of the param you define(like `/:idAndPlatform`) so to say. Instead it will just call the first endpoint it sees that is of the form 
`/api/mentions/:anyReqParams`, which could also be something like`/api/mentions/:justById`, and that could lead to unexpected behavior when debugging such as trying to call :idAndPlatform route but instead you might get :justById route and vice versa

Reply:
> maybe one is something called like /api/mentions/:justById

I think this is not possible, because the primary key of the `mention` model is just id + platform, so under no circumstance should we retrieve a mention by only id. Thus, `/api/mentions/:id` is jusy equal to `/api/mentions/:idAndPlatform`